sirepo/simulation_db.py

```diff
@@ -963,15 +1012,62 @@ def _init():
         nfs_tries=(10, int, 'How many times to poll in hack_nfs_write_status'),
         nfs_sleep=(0.5, float, 'Seconds sleep per hack_nfs_write_status poll'),
     )
+    props = pkcollections.Dict()
```

Comment:
This doesn't need to be here. It could be defined in the constant declaration. 
```py
LOADED_FILE_PROPS = [
    props=pkcollections.Dict(
        source='external',
        dir='ext',
...
```

The data structure is not the right "impedance". Rather, code